requirements.txt

```diff
@@ -10,3 +10,6 @@ crcmod==1.7
 tqdm==4.65.0
 flask==2.2.0
 flask-sock==0.6.0
+ua-parser==0.10.0
+user-agents==2.2.0
+pyyaml==5.4.1
```

Comment:
We don't seem to use pyyaml?

Reply:
Actually I need to remove both `ua-parser` and `pyyaml` as both are not being used in the implementation I went with. (I only ended up using `user-agents.parse` as I didn't need the full functionality of `ua-parser`).

Reply:
No, actually I am wrong: 

```
user-agents is hosted on [PyPI](http://pypi.python.org/pypi/user-agents/) and can be installed as such:

pip install pyyaml ua-parser user-agents
```

I'll test by removing `pyyaml` and `ua-parser` but they are included as Deps of `user-agents` so removing them from `requirements.txt` shouldn't actually prevent them from being installed (since they are in the `requirements` for `user-agents`).

Reply:
That sounds good. We should only depend on things we `import`. If those imports change their implementation to use different dependencies, that shouldn't concern us.

Reply:
This is solved ✅